src/components/CalendarDay.jsx

```diff
@@ -48,17 +48,15 @@ const defaultProps = {
   phrases: CalendarDayPhrases,
 };
 
-class CalendarDay extends React.Component {
+const BaseClass = baseClass();
+
+class CalendarDay extends BaseClass {
```

Comment:
this is going to break react linting; we'll need a jsdoc comment to indicate that it extends React.Component